android/app/src/main/res/values/dimens.xml

```diff
@@ -91,4 +91,13 @@
     <dimen name="feed_item_corner_radius">3dp</dimen>
     <dimen name="feed_item_border_width">1dp</dimen>
     <dimen name="comment_list_height">300dp</dimen>
+
+    <!--Upcoming dialogbox-->
+    <dimen name="dialog_width">300dp</dimen>
+    <dimen name="upcoming_title_height">50dp</dimen>
+    <dimen name="upcoming_height">70dp</dimen>
+    <dimen name="upcoming_margin_left">30dp</dimen>
```

Comment:
Use existing test-size, margin etc. No need to separately define this for particular view.